Meeting Notes EIPIP50 (#113)
ethereum-cat-herders/EIPIP: All EIPIP Meetings/Meeting 050.md

**ACTION 50.3**: Shashank will create an Issue to fix the validator check -> Field: Withdraw reason [23:32](https://www.youtube.com/watch?v=htwlb08kueg&t=23m32s)

**Pooja Ranjan**  00:00
Welcome to EIPIP meeting 50. I have just shared the agenda in the chat.

## AGENDA ##
## 1. EIP Editor onboarding

**Pooja Ranjan**
The first item listed here is EIP editor onboarding. So for the past few months we have been running this EIP editors apprenticeship program to onboard few more reviewers to help out with the EIP editing process. Especially on the standard track ERC proposals and the good news is we have received good responses and possible candidates who have been contributing on the standard track core and ERC proposals.
In the last meeting micah brought up that we can discuss onboarding of these candidates because some of them have been contributing for quite a few months now i know we have discussed this in the past but i suppose we can probably agree on the candidate name and decide on the next steps

**Micah Zoltu**
Like i said last time i would love to get some more editors formally on board

**Pooja Ranjan**

> Right i totally agree and i know like client you have been facilitating these meetings and from my experience one of the name is prominent over there that is william schwab so i would like to propose him to be like formally onboarded and if you have any thoughts suggestions recommendation on that

Reviewer: ```suggestion
Right i totally agree and i know Lightclient you have been facilitating these meetings and from my experience one of the name is prominent over there that is William Schwab so i would like to propose him to be like formally onboarded and if you have any thoughts, suggestions, recommendation on that?
```

**Pooja Ranjan**
oh yeah sorry my bad going forward i will be also adding about eipv issues as well so yeah if you have any update please

**Shashank Yalamanchi**
sure so i i noticed that alpha key has already picked up couple of issues and he's trying to work on the the EIP EIP hyphen bolt so i'm i'm thinking of picking up the ip validator bot end of resolve couple of issues i m i do have some questions regarding one or two issues i'm not sure if we are open to if the follower is open to ask questions regarding that will that be fine?

**Pooja Ranjan**
i hope so we have editors here so i hope your question should be answered here yeah

**Shashank Yalamanchi**
so i think the main the main one is that discussion regarding you know have we formed the consensus around that we should have the discussions to pointing to etheremagicians because i saw comments again on the EIP eipv issue that was raised some account named arachnid commented and also micah was a part of that discussion

**Micah Zoltu**

> yeah i believe we are every i believe everybody who is an active editor is an agreement this of course is a very different set of people than all the people in the world and we are not in agreement with all the people in the world and i think we are agreement with all the active editors

Reviewer: ```suggestion
 i believe everybody who is an active editor is an agreement this of course is a very different set of people than all the people in the world and we are not in agreement with all the people in the world and i think we are agreement with all the active editors
```

**Micah Zoltu**
i'm also okay with either of those solutions okay those those solutions that alex proposed are more robust than the one i proposed the one i proposed is very easy so it depends if you if you feel you have the time and capacity to implement alex's solutions i do think those are better if we have to choose between doing nothing and doing my solution then i would pick my solution 

**Shashank Yalamanchi**
sure make sense okay that's there so matt there was this issue opened called remove updated will it be fine if i share my screen so it's easy for everyone? i kind of didn't understand this EIP should not be allowed in with updated so when i open this issue it points to this one

**Micah Zoltu**
yeah so there there is a very constrained set of fields that are allowed in the header and updated is not one of them and so i don't know if the validator currently validates that there are no extraneous fields i don't think it does but it should so the validator should when it's checking should make sure that there are no fields in the header besides the ones that are official fields

**Shashank Yalamanchi**
Got it. yeah i think that'll wait thank you

**Pooja Ranjan**

> yeah i think that is from the one of the recent pull requests it's a new proposal fairly new proposal that had that additional fee and we were wondering like it would be nice to have the part validating it and i should not auto merge that

Reviewer: ```suggestion
yeah i think that is from the one of the recent pull requests it's a new proposal fairly new proposal that had that additional field and we were wondering like it would be nice to have the bot validating it and it should not auto merge that
```

**Micah Zoltu**
it's a hard problem the best chance we have at that is that when the when editor goes to review it they're the most likely ones who will have seen something similar before and they can like if they notice that this is a duplicate or something similar to another one, they'll at least i will usually tell the person hey this looks a lot like EIP blah blah blah i'm going to check that out before merging this. that being said it is okay like people are allowed to submit duplicates, in fact if there's a strong enough disagreement between two parties about how he should move forward one of the recommendations for resolution that we we give people is that they should both just create their own EIP and operate separately. and such in such case those EIPs will be duplicates of each other basically so yeah so there's no fully automated solution for this that i can think of that doesn't involve like designing a revolutionary artificial intelligence and so i think just best guess of editors or best effort of editors is all we've got

**Pooja Ranjan**

> that's yeah probably the right solution and i'm sorry i i suppose i'm set the wrong pull request number the correct one is 4671 i have shared the link in the chat here.

Reviewer: ```suggestion
that's yeah probably the right solution and I'm sorry I suppose I said the wrong pull request number the correct one is 4671. I have shared the link in the chat here.
```

> okay that sounds fair and that would be the recommendation for new editors looking into it they have finished potatoes looking into these proposals.

Reviewer: ```suggestion
okay that sounds fair and that would be the recommendation for new editors looking into it.
```
